Backend/src/dataAccessorObjects/userDAO.ts

```diff
@@ -136,7 +141,8 @@ export default class UserDataAccessorObject {
         }
         else {
           try {
-            connection.query('INSERT INTO `exam`.`users` (`username`, `password`, `email`, `isOAuth`, `refreshToken`) VALUES ( ?, ?, ?, ?, ?);',
+            // currentSchema is decided within the program and not by any user so this doesn't create a vulnerability
+            connection.query('INSERT INTO `' + this._currentSchema + '`.`users` (`username`, `password`, `email`, `isOAuth`, `refreshToken`) VALUES ( ?, ?, ?, ?, ?);',
```

Comment:
Actually I don't even think this is needed since the schema is set in the connection pool... I will test this